qutebrowser/browser/dirbrowser.py

```diff
@@ -0,0 +1,68 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+
+# Copyright 2015 Antoni Boucher (antoyo) <bouanto@zoho.com>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""The directory browser page."""
+
+import os
+import pkg_resources
+
+import qutebrowser
+from qutebrowser.utils import jinja
+
+
+def dirbrowser(url):
+    """Get the directory browser web page.
+
+    Args:
+        url: The directory path.
```

Comment:
Please call this `urlstring` - I usually assume `url` to mean a `QUrl`, so this would make it clearer.